src/main/java/com/dell/isg/smi/service/device/discovery/controller/DeviceDiscoveryController.java

```diff
@@ -55,7 +53,7 @@
 
 
     @RequestMapping(value = "/ips", method = RequestMethod.POST, headers = "Accept=application/json", consumes = "application/json", produces = "application/json")
-    @ApiOperation(value = "/ips", nickname = "ips", notes = "This operation allow user to collect server software identity throu wsman.", response = DiscoverdDeviceResponse.class, responseContainer = "List")
+    @ApiOperation(value = "/ips", nickname = "ips", notes = "This operation ping sweep and discovers the devices within the given list of IP.", response = DiscoverdDeviceResponse.class, responseContainer = "List")
```

Comment:
Change to:
This operation will ping sweep and discover devices within the given IP list.

Reply:
Done